modules/client-configuration/pages/supported-features.adoc

```diff
@@ -36,83 +36,16 @@ For details on supported product versions, see https://www.suse.com/lifecycle.
 | {sle} 12 | {x86}_64, {ppc}, {ibmz}, ARM               | icon:check[role="green"]      | icon:check[role="green"]
 | {sle} 11 | {x86}, {x86}_64, {ipf}, IBM {ppc}, {ibmz}  | icon:check[role="green"]      | icon:check[role="green"]
 | {sles} for SAP | {x86}_64, {ppc}                       | icon:check[role="green"]      | icon:check[role="green"]
-| {sles}-ES 8 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
+| {opensuse} Leap 15.1 | {x86}_64                       | icon:times[role="danger"]      | icon:check[role="green"]
+//| {sles}-ES 8 | {x86}_64                                 | icon:check[role="danger"]      | icon:check[role="green"]
 | {sles}-ES 7 | {x86}_64                                 | icon:check[role="green"]      | icon:check[role="green"]
 | {sles}-ES 6 | {x86}, {x86}_64                          | icon:check[role="green"]      | icon:check[role="green"]
-| {rhel} 8 | {x86}_64                                   | icon:question[role="gray"]    | icon:question[role="gray"]
+//| {rhel} 8 | {x86}_64                                   | icon:question[role="danger"]    | icon:question[role="gray"]
 | {rhel} 7 | {x86}_64                                   | icon:check[role="green"]      | icon:check[role="green"]
 | {rhel} 6 | {x86}, {x86}_64                            | icon:check[role="green"]      | icon:check[role="green"]
-//| CentOS 7 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
-//| CentOS 6 | {x86}, {x86}_64                            | icon:question[role="gray"]    | icon:question[role="gray"]
-| {opensuse} Leap 15.1 | {x86}_64                       | icon:times[role="danger"]      | icon:check[role="green"]
-| {ubuntu} 16.04 | {x86}_64                             | icon:times[role="danger"]      | icon:check[role="green"]
+| {centos} 7 | {x86}, {x86}_64                            | icon:question[role="gray"]/icon:check[role="green"] (with ES)   | icon:question[role="gray"] / icon:check[role="green"] (with ES)
+| {centos} 6 | {x86}, {x86}_64                            | icon:question[role="green"]    | icon:question[role="gray"]
```

Comment:
CentOS6 with ES is supported AFAIK. Keeping in mind the EoL of CentOS6, I will mark it with a red X, instead of questions.

Reply:
That's not listed in our key. I would go for a red X in this case.

Reply:
According to https://wiki.centos.org/About/Product#fndef-a91b3c0c287c782f9af063daff9e64b566d648c7-1 CentOS 6 still receives maintenance updates.

Reply:
Until Nov 2020, but I am nnot alking about that.

What I say is that a question mark means that maybe we'll evaluate support as OS for the future. And I am saying:

"Since CentOS6 goes EoL on Nov 2020, I don't think we'll focus on supporting it as base OS, and we should mark it as 'not supported'" :-)